MapboxAndroidDemo/src/global/java/com/mapbox/mapboxandroiddemo/MapboxApplication.java

```diff
@@ -46,4 +38,12 @@ private void setUpTileLoadingMeasurement() {
       .build();
     HttpRequestUtil.setOkHttpClient(okHttpClient);
   }
+
+  private void setUpCrashlytics() {
+    Fabric.with(this, new Crashlytics.Builder()
+        .core(new CrashlyticsCore.Builder()
+            .disabled(BuildConfig.DEBUG)
```

Comment:
We are configuring not collecting any data for debug builds in https://github.com/mapbox/mapbox-android-demo/pull/1104/files#diff-5f14090300b04edf8f7f1a0190d2872aR76. Do we need to keep both configurations?

Reply:
Yes, seems that it's best to have both configurations 

- https://docs.fabric.io/android/crashlytics/build-tools.html#disable-crashlytics-for-debug-builds
- https://stackoverflow.com/a/28357998/6358488
- https://stackoverflow.com/a/16990788/6358488